### PR TITLE
Upgrade ES from 5.5 to 5.6.11

### PR DESCRIPTION
**Pull request by bio-boris**
Upgrade ES from 5.5 to 5.6.11

### CODE REVIEW

**Comment by bio-boris** [comment]
Note the one error and warnings are already in the current builds

https://travis-ci.org/kbase/KBaseSearchEngine/jobs/427889488#L1656

https://travis-ci.org/kbase/KBaseSearchEngine/jobs/427889488#L1669-L1671

(You might need to click the link again in order to get to the line number, or refresh the page)

**Comment by MrCreosote** [comment]
@tiramisu24 can you review this one please?

File: lib/src/kbasesearchengine/search/ElasticIndexingStorage.java [hunk 1]
```diff
@@ -726,7 +726,7 @@ private int updateLastVersionsInData(String indexName, GUID parentGUID,
         // script = {"inline": "ctx._source.islast = (ctx._source.version == params.lastver)",
```
Comment (tiramisu24) [inline lib/src/kbasesearchengine/search/ElasticIndexingStorage.java]: comments should be changed to source too

File: lib/src/kbasesearchengine/search/ElasticIndexingStorage.java [hunk 2]
```diff
@@ -726,7 +726,7 @@ private int updateLastVersionsInData(String indexName, GUID parentGUID,
...
         //           "params": {"lastver": lastVersion}}
         Map<String, Object> script = ImmutableMap.of(
-                "inline", "ctx._source.islast = (ctx._source.version == params.lastver);",
```
Comment (tiramisu24) [inline lib/src/kbasesearchengine/search/ElasticIndexingStorage.java]: is inline to source a breaking change?

Comment (tiramisu24) [reply lib/src/kbasesearchengine/search/ElasticIndexingStorage.java]: actually is this change necessary? I don't see it in breaking changes from 5.5 to 5.6
https://www.elastic.co/guide/en/elasticsearch/reference/5.6/breaking-changes-5.6.html

File: lib/src/kbasesearchengine/search/ElasticIndexingStorage.java (hunk 1 above)
Comment (bio-boris) [reply lib/src/kbasesearchengine/search/ElasticIndexingStorage.java]: done

File: lib/src/kbasesearchengine/search/ElasticIndexingStorage.java (hunk 2 above)
Comment (bio-boris) [reply lib/src/kbasesearchengine/search/ElasticIndexingStorage.java]: The change is not necessary, but it makes all of the warnings disappear and thus the logs are not flooded.

**Pull request merged**